docs/_user_guide/data_transfer.md

```diff
@@ -1,23 +1,170 @@
+On your local machine, in a terminal, run:
+
+```title="Local Terminal Command"
+$ touch my_file.txt
+$ scp my_file.txt <SUNetID>@yen.stanford.edu:~
+```
+where `~` is your Yen home directory shortcut. Enter your SUNet ID password and Duo authenticate for the file transfer to complete.
+
+#### Transferring Folders to Yen Servers
+On your local machine, open a new terminal and navigate to the parent directory of the folder that 
+you want to transfer to the Yens with the `cd` command.
+
+Once you are in the parent directory of the folder you want to transfer, run the following to copy the folder to the Yens:
+
+```title="Local Terminal Command"
+$ scp -r my_folder/ <SUNetID>@yen.stanford.edu:/zfs/projects/students/<my_project_dir>
+```
+The `-r` flag is used to copy folders (**r**ecursively copy files). Replace `<SUNetID>` with your SUNet ID and `<my_project_dir>` with the destination path on the Yen's file system, ZFS. 
+
+Let's illustrate this with an example. We'll create an empty folder called `test_from_local` and transfer it to the home directory on Yen servers:
+
+```title="Local Terminal Command"
+$ mkdir test_from_local
+$ scp -r test_from_local/ <SUNetID>@yen.stanford.edu:~
+```
+
+#### Transferring Files from Yen Servers
+When transferring data from the Yens to your local machine, the `<source_path>` is the path to the file(s) on the Yens while the `<destination_path>` is the path on your local machine where the files from the Yens will be transferred to.
+
+To copy files from Yen servers to your local machine, open a new terminal without connecting to the Yens. Use the `cd` command to navigate to the local directory where you want to copy the files to. Then run: 
+
+```title="Local Terminal Command"
+$ cd my_local_folder
+$ scp -r <SUNetID>@yen.stanford.edu:/zfs/projects/students/<my_project_dir>/results .
+```
+
+In this example, we're copying the `results` folder from the Yen's ZFS file system to your local directory ( ++period++ signifies the current directory). If you're copying files (not directories), omit the `-r` flag. To transfer multiple files, use the wildcard `*` to match several files.
+
+
+## Rsync
+Another option for more reliable transfers is `rsync`. `rsync` is a utility for efficiently transferring and synchronizing files between two locations. It is faster than `scp` because it only copies the differences between files. This means that if a file is partially transferred, `rsync` will only copy the remaining part of the file, rather than retransferring the entire file.
+
+
+#### Transferring Files to Yen Servers
+To transfer a file (e.g., `myfile.csv`) from your local machine, use:
+
+```title="Terminal Command"
+$ rsync -aP myfile.csv <SUNetID>@yen-transfer.stanford.edu::/zfs/projects/students/<my_project_dir>
+```
+You'll be prompted to enter your password and complete the two-step authentication process.
+
+#### Transferring Folders to Yen Servers
+To transfer a folder, add the recursive flag (`-r`) to `rsync`:
```

Comment:
No, you don't need -r if you're already using the -a flag in rsync. Here's why:

The -a flag (archive mode) is a comprehensive option that includes recursive behavior (equivalent to -r) as well as additional features such as preserving symbolic links, file permissions, timestamps, and ownership. Using -a alone is sufficient for most file and folder transfers.

When Might You Use -r Separately?
You would only use -r by itself (or alongside other flags) if you specifically want recursive behavior but don’t need the additional features of -a. For example, if you don’t care about preserving metadata like timestamps or permissions, you might use -r instead of -a.

Redundancy in -aPr
In your commands like rsync -aPr, the -r is redundant because -a already includes recursive functionality. You can simplify it to rsync -aP.